Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo ()
+        {
+            string userFirstName = UserName(message: "Enter your first name: ");
+            string userSurname = UserName(message: "Enter your surname: ");
+            int userAge = UserAge(message: "Enter your age: ");
+            float userHeight = UserHeightWeight(message: "Enter your height in cm: ");
+            float userWeight = UserHeightWeight(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
```

Comment:
Those function names are reflecting what you want to do with them but the functionality differs from that. Imagine you want to reuse UserName() to request the name of his town or UserAge() to ask for the amount of siblings the user has.
This won't work and therefor the choice of names here is bad. Please fix! :)